daemon/note.go

```diff
@@ -1,4 +1,4 @@
-package git
+package daemon
```

Comment:
`Note` can be unexported.

Github won't let me comment on an unchanged but moved file, because nothing notable could possibly happen when moving a file...

Reply:
Yes, and so (elsehwere) can `daemonJobFunc`.